packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -162,18 +162,61 @@ function getCompilerConfig(wco: WebpackConfigOptions): webpack.Configuration {
   return {};
 }
 
+interface I18nOptions {
+  sourceLocale?: string;
+  locales: Record<string, { file: string; translation: unknown }>;
+}
+
 async function initialize(
   options: BrowserBuilderSchema,
   context: BuilderContext,
   host: virtualFs.Host<fs.Stats>,
   webpackConfigurationTransform?: ExecutionTransformer<webpack.Configuration>,
-): Promise<{ config: webpack.Configuration[]; projectRoot: string; projectSourceRoot?: string }> {
+): Promise<{
+  config: webpack.Configuration[];
+  projectRoot: string;
+  projectSourceRoot?: string;
+  i18n: I18nOptions;
+}> {
   const { config, projectRoot, projectSourceRoot } = await buildBrowserWebpackConfigFromContext(
     options,
     context,
     host,
   );
 
+  // target is verified in the above call
+  // tslint:disable-next-line: no-non-null-assertion
+  const metadata = await context.getProjectMetadata(context.target!);
+
+  const i18n: I18nOptions = { locales: {} };
+
+  if (metadata.sourceLocale !== undefined && typeof metadata.sourceLocale !== 'string') {
+    throw new Error('Project i18n sourceLocale field is malformed.  Expected an string.');
+  }
+  // en-US is the default locale added to Angular applications (https://angular.io/guide/i18n#i18n-pipes)
+  i18n.sourceLocale = metadata.sourceLocale || 'en-US';
+
+  if (
+    metadata.locales !== undefined &&
+    (!metadata.locales || typeof metadata.locales !== 'object' || Array.isArray(metadata.locales))
+  ) {
+    throw new Error('Project i18n locales field is malformed.  Expected an object.');
+  } else if (metadata.locales) {
+    for (const [locale, translationFile] of Object.entries(metadata.locales)) {
+      if (typeof translationFile !== 'string') {
+        throw new Error(
+          `Project i18n locales field value for '${locale}' is malformed.  Expected an string.`,
```

Comment:
`Expected an string` -> `Expected a string`.

Also before `Expected` there seems to be a double space. Same for `Expected an object `

Reply:
Done.  Fixed all occurrences.